src/managers/SaveUpdateManager.ts

```diff
@@ -0,0 +1,51 @@
+/*
+Save files are versioned by incrementing integer numbers. These numbers correspond 
+to the length of the transformers array and the version ONLY increments when a new 
+transformer is written.
+
+The current version is always equal to transformers.length, and each entry 
+in transformers gets one step closer to the current version (ie, "toOne" transforms 
+version 0 save files [pre-versioning save files] to version 1 save files.)
+
+If a save file is multiple versions behind, it is run in order against each 
+sequential transformer that applies to it. ie a version 2 savefile updating to 
+version 5 will be run against toThree, toFour, and toFive, or in other words,
+entries 2, 3, and 4 in the array, which would be the final three entries.
+
+Functions are defined separately as toOne/toTwo/etc as a sanity check. 
+If using just a bare array, it would be too easy to accidentally switch two transformers 
+and not even know, especially while refactoring. By naming them, it becomes
+much more difficult to get them out of order and much easier to fix if they do.
+*/
+
+/**
+ * Updates response bodies to be strings, rather than Record<string, unknown>.
+ */
+function toOne(data: any) {
```

Comment:
Right now, when we load dates from the db, we transform them from `Date` strings to numbers. Should we also move that logic to this transformer?

Reply:
Do we want them to be stored as numbers in the future? This edits the data so that when it's saved again it'll be in the correct format.